bundles/org.eclipse.swt/Eclipse SWT Custom Widgets/common/org/eclipse/swt/custom/CTabFolder.java

```diff
@@ -3470,6 +3471,22 @@ public void setSelectionForeground (Color color) {
 	if (selectedIndex > -1) redraw();
 }
 
+/**
+ * Sets the thickness of the highlight bar on the selected tab. The highlight bar is drawn in the top margin of the selected tab.
+ *
+ * @param thickness the desired thickness. Most be positive and lower than {@link CTabFolderRenderer#ITEM_TOP_MARGIN} (that is either {@code 0} {@code 1} or {@code 2} at the moment),for correct results.
+ *                  Set to {@code 0} to not draw a highlight bar.
+ * @implNote Default {@link CTabFolderRenderer} currently ignores this setting if {@link #getSimple()} is {@code true}.
+ * @since 3.121
```

Comment:
New API Javadoc need to call-out the exception:
<details>
<summary>Snippet</summary>

```java
 * @exception SWTException <ul>
 *    <li>ERROR_INVALID_ARGUMENT - if called with a thickness of less than 0</li>
 * </ul>
 *
 * @since 3.121
 */
```
</details>